app/browser/api/ledger.js

```diff
@@ -2160,7 +2167,13 @@ const transitionWalletToBat = () => {
   }
 
   if (client.busyP()) {
-    console.log('ledger client is currently busy; transition will be retried on next launch')
+    if (++busyRetryCount > 3) {
+      console.log('ledger client is currently busy; transition will be retried on next launch')
```

Comment:
should we hide transition overlay when busy fails for the third time?

Reply:
I don't know if it makes sense to hide it without providing a work-around. We don't want to encourage using the old servers indefinitely- I think it would be better to update the transition screen to provide a community link about stuck transitions.

cc: @alexwykoff